src/panel/TemplateEditor/lib/cloud/portal.ts

```diff
@@ -0,0 +1,13 @@
+
+import * as request from "./request";
+
+interface PortalParams {
+    params?: Record<string, unknown>;
+}
+
+export const load = async (params: PortalParams, userId?: string) => {
```

Comment:
Should we move this, and `getUser` and other 'new' network calls into syphonx-lib?